incubator/hnc/doc/metrics/stackdriver.md

```diff
@@ -0,0 +1,85 @@
+# How to View HNC Metrics in Stackdriver
```

Comment:
Maybe "Stackdriver on GKE?" I think SD is usable from other clouds as well. Also rename the file to `stackdriver-gke.md`.